tests/services/kernels/test_api.py

```diff
@@ -240,3 +250,28 @@
 async def test_config2(serverapp):
     assert serverapp.kernel_manager.allowed_message_types == []
 
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 6),
+    reason="Kernel manager is AsyncMappingKernelManager, Python version < 3.6"
+)
+async def test_async_kernel_manager(configurable_serverapp):
+    argv = ['--ServerApp.kernel_manager_class=jupyter_server.services.kernels.kernelmanager.AsyncMappingKernelManager']
+    app = configurable_serverapp(argv=argv)
+    assert isinstance(app.kernel_manager, AsyncMappingKernelManager)
+
+
+@pytest.fixture()
+def no_async_kernel_manager(monkeypatch):
+    """Remove jupyter_client.multikernelmanager.AsyncMultiKernelManager."""
+    monkeypatch.delattr("jupyter_client.multikernelmanager.AsyncMultiKernelManager")
+    monkeypatch.delattr("jupyter_server.services.kernels.kernelmanager.AsyncMappingKernelManager")
```

Comment:
`AsyncMappingKernelManager` will always exist.  This should only remove `AsyncMultiKernelManager` to simulate older jupyter_clients.

Reply:
Done.